graphql_compiler/query_pagination/pagination_planning.py

```diff
@@ -22,6 +23,8 @@
 )
 
 
+# TODO(bojanserafimov): Make this function return a best effort pagination plan
+#                       when a good one is not found instead of returning None.
 def get_pagination_plan(schema_info, query_ast, number_of_pages):
```

Comment:
nit: How about `try_get_pagination_plan`? Also, update the docstring to reflect this.